exe-common/src/network/ntt.rs

```diff
@@ -0,0 +1,73 @@
+use futures::Future;
+use tokio::runtime::Runtime;
+
+use network::api::{Api, BlockRef};
+use network::{Error, Result};
+
+//to_socket_addr
+use network_core::client::block::HeaderService;
+use network_ntt::client as ntt;
+use std::net::SocketAddr;
+
+use cardano::{
+    block::{Block, BlockHeader, HeaderHash, RawBlock},
+    tx::{TxAux, TxId},
+};
+
+pub struct NetworkCore {
```

Comment:
Why is this named `NetworkCore`?

Reply:
I was planning to have a single structure that will keep runtime for all the threads, but that didn't work out. Any better name suggestion will be appreciated.